### PR TITLE
Friday sprint half 2

### PR DESCRIPTION
**Pull request by brisvag**

### CODE REVIEW

File: walkthroughs/EMPIAR-10164/m.md [hunk 1]
```diff
@@ -1,4 +1,60 @@
 # Multi-particle refinement
 
-Multi particle refinement in M.
-## Black Magic
+Now that our particle positions and orientations are mostly correct, there are several parameters that can be tweaked _a posteriori_ to refine the result, such as defocus, inter-frame and tilt-induced motions, and more.
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/m.md:3]: Now that our particle poses are mostly correct and we have a good estimate of the reconstructed density, there are several parameters that can be tweaked _a posteriori_ to further refine our result. We will perform these optimisations in M.

File: walkthroughs/EMPIAR-10164/m.md [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+
+````{margin}
+```{seealso}
+We highly recommend [the `M` paper](https://doi.org/10.1101/2020.06.05.136341) to better understand what this software does and is capable of.
+```
+````
+
+`M` provides an automated, iterative procedure for concerted optimisation of these parameters.
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/m.md:11]: `M` provides a procedure for the concerted optimisation of these parameters.

File: walkthroughs/EMPIAR-10164/m.md [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+Note that this time we are not adding a soft edge: `M` wants a binary map!
+```
+````
+Open one of the halfmaps in your favourite program and determine an appropriate threshold for binarisation. Use this as the `--ini_threshold` parameter in the following command, replacing `<latest_halfmap>` as appropriate:
+
+```bash
+relion_mask_create --i <latest_halfmap>.mrc --angpix 1.6 --extend_inimask 5 --o mask_1.6Apx.mrc --ini_threshold 0.05
+```
+
+Provide this map to `M`, and then select the final particle positions (e.g: `run_it019_data.star`) for the particle coordinates. Click `FINISH` to generate the new species.
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/m.md:36]: I forgot to mention, we used 3 temporal samples for particle poses on import and only the first 15 tilt images, this is important

File: walkthroughs/EMPIAR-10164/m.md [hunk 4]
```diff
@@ -1,4 +1,60 @@
...
+
+First, run four iterations with these parameters:
+![refinement1](m.assets/refinement1.png)
+
+Afterwards, three iterations as follows:
+![refinement2](m.assets/refinement2.png)
+
+## End Result!
+
+If everything went well, after the second refinement you should be able to get to something like this:
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/m.md:54]: after the second round of refinements

File: walkthroughs/EMPIAR-10164/particle-picking.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ The provided scripts are just a few lines long, and are given for convenience an
 
 To isolate a good subset of particles in lattices we will use three short scripts provided with this tutorial. 
 
-The first cleaning step we should take is removing duplicate particles. To do so, we will use a simple matlab script, [`remove_duplicates.m`](scripts/remove_duplicates.m).
+The first cleaning step we should take is removing duplicate particles. To do so, we will use a simple matlab script, [`remove_duplicates.m`](https://github.com/open-subtomo/open-subtomo/tree/master/walkthroughs/EMPIAR-10164/scripts/remove_duplicates.m).
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/particle-picking.md:44]: this needs to be changed to the teamtomo repo

File: walkthroughs/EMPIAR-10164/particle-picking.md [hunk 6]
```diff
@@ -66,7 +66,7 @@ dwrite(nodup, 'result_10Apx_nodup.tbl');
 
 This will create a new table called `result_10Apx_nodup.tbl`, reducing clusters of multiple particles within 4 pixels (half the distance we measured earlier) to a single particle.
 
-The next step is to remove particles that don't belong to the lattice. To do so, we use [`check_radial_distribution.m`](scripts/check_radial_distribution.m).
+The next step is to remove particles that don't belong to the lattice. To do so, we use [`check_radial_distribution.m`](https://github.com/open-subtomo/open-subtomo/tree/master/walkthroughs/EMPIAR-10164/scripts/check_radial_distribution.m).
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/particle-picking.md:69]: teamtomo repo

File: walkthroughs/EMPIAR-10164/particle-picking.md [hunk 7]
```diff
@@ -111,7 +111,7 @@ If you are wondering why the second peak has two sub-peaks: it's the honeycomb p
 
 ![radial distribution](particle-picking.assets/radial-distance-profile.png)
 
-This confirms the previously measured interparticle distance and the fact that most particles slid into ordered positions in the lattice. We can use the first peak to select against particles that don't conform to this distribution. The next script, [`subset_table_based_on_neighbours.m`](scripts/subset_table_based_on_neighbours.m), removes all particles that don't have at least 3 neighbours in the radial shell at distance 7.5 $px$. To use it, run:
+This confirms the previously measured interparticle distance and the fact that most particles slid into ordered positions in the lattice. We can use the first peak to select against particles that don't conform to this distribution. The next script, [`subset_table_based_on_neighbours.m`](https://github.com/open-subtomo/open-subtomo/tree/master/walkthroughs/EMPIAR-10164/scripts/subset_table_based_on_neighbours.m), removes all particles that don't have at least 3 neighbours in the radial shell at distance 7.5 px. To use it, run:
```
Comment (alisterburt) [inline walkthroughs/EMPIAR-10164/particle-picking.md:114]: teamtomo repo

**Review by alisterburt — COMMENTED** [review]
excellente! looking great, a few little bits that need addressing then we're good to go!

**Comment by brisvag** [comment]
Ok, I addressed everything, plus a bit more (there were a bunch more references to open-subtomo all over the repo)

**Pull request merged**